include/process.h

```diff
@@ -87,7 +87,10 @@ class Thread
     void suspend();
     void resume();
 
-    static Thread * volatile self() { return running(); }
+    static Thread * volatile self();
+
+    static unsigned int cpu_id() { return CPU::id() + 1; } // For identifier the cpu id in tests
```

Comment:
```suggestion
    static unsigned int cpu_id() { return CPU::id(); } // For identifier the cpu id in tests
```
Se for lidar com 1 em tudo não precia disso aqui